components/application-mgt/org.wso2.carbon.identity.application.mgt.ui/src/main/java/org/wso2/carbon/identity/application/mgt/ui/client/ApplicationManagementServiceClient.java

```diff
@@ -194,7 +194,7 @@ public IdentityProvider getFederatedIdentityProvider(String identityProviderName
         try {
             idps = stub.getAllIdentityProviders();
         } catch (Exception e) {
-            e.printStackTrace();
+            log.error(e.getMessage(),e);
```

Comment:
don't log "e.getMessage()", It's logged with "e" anyway. Log a message with the context information